tests/ceph_installer/test_ansible.py

```diff
@@ -69,8 +69,15 @@ def run(ceph_cluster, **kw):
 
     log.info("Ceph ansible version " + ceph_installer.get_installed_ceph_versions())
 
+    # ansible playbookk based on container or bare-metal deployment
+    file_name = "site.yml"
+
+    if ceph_cluster.containerized:
+        file_name = "site-container.yml"
+
     out, rc = ceph_installer.exec_command(
-        cmd='cd {} ; ANSIBLE_STDOUT_CALLBACK=debug;ansible-playbook -vvvv -i hosts site.yml'.format(ansible_dir),
+        cmd='cd {ansible_dir} ; ANSIBLE_STDOUT_CALLBACK=debug;ansible-playbook -vvvv -i hosts {file_name}'
```

Comment:
would suggest to have `cmd` string framed outside calling function, so that would be more readable.